packages/FormElement/src/components/Help/Help.js

```diff
@@ -1,42 +1,49 @@
 import React from "react";
 import PropTypes from "prop-types";
-
-import InfoCircleIcon from "@paprika/icon/lib/InfoCircle";
 import Popover from "@paprika/popover";
 import useI18n from "@paprika/l10n/lib/useI18n";
+import * as sc from "./Help.styles";
+
+function Help(props) {
+  const { children, isDisabled, a11yText } = props;
+  const I18n = useI18n();
 
-import helpStyles, { StyledTrigger, iconStyles } from "./Help.styles";
+  return (
+    <sc.Help align="bottom" data-pka-anchor="form-element.help">
+      {isDisabled ? (
```

Comment:
Oh, I remember now that I want to be able to use the `<Label>` without the context provider (just for styling) sometimes.  Since the `<Help>` isn't used on its own, and `<Label>` gets `isDisabled` from context already, I think it's okay that this component just has the prop.

But you make a good point, and the `<Label>` should have an `isDisabled` prop, in case it's being used without context and should be disabled. The prop will take precedence if both exist.